src/UserService/Program.cs

```diff
@@ -12,16 +12,26 @@ public static void Main(string[] args)
         {
             var configuration = new ConfigurationBuilder()
                 .AddJsonFile("appsettings.json")
-#if DEBUG
-                .AddJsonFile("appsettings.Development.json")
-#else
-                .AddJsonFile("appsettings.Production.json")
-#endif
                 .Build();
 
+            string seqServerUrl = Environment.GetEnvironmentVariable("seqServerUrl");
+            if (string.IsNullOrEmpty(seqServerUrl))
+            {
+                seqServerUrl = configuration["Serilog:WriteTo:1:Args:serverUrl"];
+            }
+
+            string seqApiKey = Environment.GetEnvironmentVariable("seqApiKey");
+            if (string.IsNullOrEmpty(seqApiKey))
+            {
+                seqApiKey = configuration["Serilog:WriteTo:1:Args:apiKey"];
+            }
+
             Log.Logger = new LoggerConfiguration().ReadFrom
                 .Configuration(configuration)
                 .Enrich.WithProperty("Service", "UserService")
+                .WriteTo.Seq(
+                    serverUrl: seqServerUrl,
+                    apiKey: seqApiKey)
```

Comment:
Why we have 2 arguments, but in appsettings 3

Reply:
Spartak said that the 3rd argument is's nessesary